README.md

```diff
@@ -1,12 +1,11 @@
-daml-dit-ddit
-====
+# daml-dit-ddit
 
 `ddit` is a command line tool, written in Python, to streamline and
 automate the process of building composite artifacts for
-[project:DABL](https://www.projectdabl.com/). DABL stores composite
+[Daml Hub](https://www.projectdabl.com/). Daml Hub stores composite
```

Comment:
Shouldn't the link target be changed too?

Reply:
It sure should! Thanks